Source/Scene/GltfLoaderUtil.js

```diff
@@ -188,4 +191,62 @@ GltfLoaderUtil.createSampler = function (options) {
   });
 };
 
+var defaultScale = new Cartesian2(1.0, 1.0);
+
+/**
+ * Create a model texture.
+ *
+ * @param {Object} options Object with the following properties:
+ * @param {Object} options.textureInfo The texture info JSON.
+ * @param {String} [options.channels] The texture channels to read from.
+ * @param {Texture} [options.texture] The texture object.
+ *
+ * @returns {ModelComponents.Texture} The texture.
+ */
+GltfLoaderUtil.createModelTexture = function (options) {
+  options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+  var textureInfo = options.textureInfo;
+  var channels = options.channels;
+  var texture = options.texture;
+
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("options.textureInfo", textureInfo);
+  //>>includeEnd('debug');
+
+  var texCoord = defaultValue(textureInfo.texCoord, 0);
+  var transform;
+
+  var textureTransform = defaultValue(
+    textureInfo.extensions,
+    defaultValue.EMPTY_OBJECT
+  ).KHR_texture_transform;
+
+  if (defined(textureTransform)) {
```

Comment:
It would be good to use the `has3DTilesExtension` function here.

Reply:
Maybe `has3DTilesExtension` should be renamed to `hasExtension` so it applies equally to 3D Tiles and glTF?

(Though not to be confused with gltf-pipeline `hasExtension` which should probably be renamed to `usesExtension`.

Reply:
I'll do both renamings.

Reply:
https://github.com/CesiumGS/gltf-pipeline/pull/590 -- opened this PR for the rename in gltf-pipeline. This should be merged first and then I'll update this PR with the new files.

Reply:
I ended up having to patch the gltf-pipeline files in place, copying gltf-pipeline into `cesium` will break legacy texture compression until KTX2 is finished.